### PR TITLE
Add explicit node version metadata

### PR DESCRIPTION
**Pull request by kennythepark**
Adding a `.node-version` file, which is used by some node managers but mostly acts as more of a reference for the contributors to this package. Note, this is advisory, no coercing.

~Note: Choosing Node v16 (Gallium) as it is the current [active LTS version](https://nodejs.org/en/about/releases/).~ See comments below.

### CODE REVIEW

**Comment by danielcolinjames** [comment]
Screen Shot 2021-11-13 at 4 51 21 PM
adding you to the vercel but in the meantime here's the build error

**Comment by danielcolinjames** [comment]
seems like v14 is the latest supported by vercel right now, so I guess we can use that for now

**Comment by danielcolinjames** [comment]
are you familiar with yarn btw @kennythepark? I just tried `npm i && npm run dev` on this branch and got a weird error that I think I've actually seen before on the Footer.js file, but I don't get that error when running the equivalent `yarn && yarn dev` for some reason 🤔   it's about the styled-jsx package and something about a config file

my understanding is that yarn is just a wrapper around npm (made by fb) and the only real developer experience differences are: it generates yarn.lock instead of package-lock.json, and supposedly runs a bit faster (anecdotally I find this to be true), and the different CLI commands

if you're down to use yarn for this project I think we might as well stick with it for now since it seems something breaks when using npm (at least on my side). but I'm also down to figure out the conflict and use npm, not a dealbreaker, I just have muscle memory for yarn CLI commands instead of npm commands now haha

**Comment by kennythepark** [comment]
- Good call, let's stick with v14 for now. 
- As for `yarn`, have not used it, but more than happy to stick to one. One caveat I found with `vercel` is that if a `package-lock.json` file exists, the build process resorts to [using `npm`, not `yarn`](https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-dependencies). I can verify that by the warning message below in our build logs:
```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. 
It is advised not to mix package managers in order to avoid resolution inconsistencies caused by 
unsynchronized lock files. To clear this warning, remove package-lock.json.
```
So, if we want to stay the `yarn` route, I'll add `package-lock.json` as part of our git ignores so that we don't get surprise build failures during deployment 👍

**Comment by danielcolinjames** [comment]
ah yeah was noticing that message too, yeah adding package-lock to the git ignore sounds good 👍  if you're okay with keeping it yarn for now :)

**Comment by danielcolinjames** [comment]
do you wanna add that to this PR? then I think this should be good to go 🚢

**Review by danielcolinjames — APPROVED** [review]
🚀

File: .gitignore [hunk 1]
```diff
@@ -35,6 +35,9 @@ yarn-error.log*
 # vercel
 .vercel
 
+# vercel - force build via yarn by removing package-lock.json
+package-lock.json
```
Comment (danielcolinjames) [inline .gitignore:39]:
```suggestion
/ath.ooo/package-lock.json
```

Comment (kennythepark) [reply .gitignore]: Does this pertain to just the `ath.ooo` project or all sub-projects in this monorepo now? I see that `ath-og-generator` is built on vercel too :)

Comment (danielcolinjames) [reply .gitignore]: sorry I just committed it right before I saw your message haha. so the way the monorepos work from vercel is you just tell it to only look at one subfolder and deploy from there (in this case, /ath.ooo/), and then git (and gitignore) are keeping track of the superfolder, so to gitignore something in any of the 3 "subrepos" we'll have to do it like that, where we specify _which_ `package-lock.json` file we're referring to (or whatever file)

Comment (danielcolinjames) [reply .gitignore]: at least I'm pretty sure that's how it all works, but I could be wrong or missing something. so I assume the gitignore is just so you can use npm if you want and it won't mess up the builds by committing package-locks to the remote repo

Comment (kennythepark) [reply .gitignore]: that's interesting, i'd be down to investigate this further but for now no need to delve in deeply 👍

**Pull request merged**